src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -306,6 +427,8 @@ export default function Transaction({ hash: id }: { hash: string }) {
   const onCancelClick = () => setShowCancelModal(true)
   const onDismiss = () => setShowCancelModal(false)
 
+  // const isLoading = isPending || isCancelling ? true : false
+
```

Comment:
Something I added, to indicate a 'transaction/card' is loading. To then trigger a shimmer/loading effect with CSS. But left it out because of priority. I can leave this in as a Todo, or?